bin/trace_file_list

```diff
@@ -1,2 +1,3 @@
 1
-../sst-unit-test/traces/cachesize_1/trace.txt
+/home/common/intel_cpu_pin/pin-3.7-97619-g0d0c92f4f-gcc-linux/source/tools/x86_trace_generator/trace.txt
```

Comment:
these new traces are not part of this repo. use the one in sst-unit-test

Reply:
resolved